src/EventStore.Projections.Core.Tests/Services/v8/when_running_a_v8_projection_emitting_malformed_links.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
```

Comment:
I can't tell the difference between these two tests

Reply:
See comment above.